query-node/schema.graphql

```diff
@@ -386,3 +386,22 @@ type FeaturedVideo @entity {
   "Reference to a video"
   video: Video!
 }
+
+enum StorageProviderType {
+  GATEWAY
+  STORAGE
+}
+
+type StorageProvider @entity {
+  "Runtime entity identifier (EntityId)"
```

Comment:
What is an entity and EntityId ?

Reply:
Do you mean runtime `WorkerId` type

Reply:
The commentaries have been improved for various IDs. Now they state `Runtime identifier` for ids that correspond to runtime ids, or `Unique identifier` where they don't correspond.